tests/testsuite/check_cfg.rs

```diff
@@ -521,6 +521,81 @@ fn lint_table_config() {
         .run();
 }
 
+#[cargo_test(>=1.79, reason = "--check-cfg was stabilized in Rust 1.79")]
+fn lint_table_config_and_features() {
```

Comment:
could you be more clear in the test name what you are trying to test here.  Is it that `lint_config doesn't cargo native check-cfg's?

Reply:
(ditto for the other tests)

Reply:
Also,. if thats the case, should we have `build.rs` / lint config tests?

Reply:
I've renamed most tests to be a more descriptive. I add more description is desired.